### PR TITLE
Drop BCD table from @scroll-timeline article

### PR DESCRIPTION
**Pull request by sideshowbarker**
This is a follow-up to https://github.com/mdn/content/pull/19056

https://github.com/w3c/csswg-drafts/commit/fc6766d dropped the scroll-timeline at-rule from the “Scroll-linked Animations” spec

The feature was only ever implemented in Firefox, and only behind a flag, and given that it’s been dropped from the spec, it seems very unlikely to ever be shipped (un-flagged) in Firefox (or in any other browser).

### CODE REVIEW

File: files/en-us/web/css/@scroll-timeline/index.md [hunk 1]
```diff
@@ -166,10 +166,6 @@ We create an `@scroll-timeline` called `squareTimeline`, setting the `source` as
 
 {{EmbedLiveSample("Simple_example")}}
```
Comment (teoli2003) [inline files/en-us/web/css/@scroll-timeline/index.md:168]: We need to tell the reader that this is not part of a standard and that no browser implemented it. Else the devs will believe this is something that can be used.

(I even wonder if we shouldn't just delete the page, but I'm not sure)
```suggestion

## Specifications

This feature was in a draft specification and has been removed from it. It is no longer on track to become a standard.

## Browser compatibility

No released browser ever implemented this feature.
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]

**Comment by queengooborg** [comment]
If no browser ever implemented this and the specification was dropped, why don't we drop this page entirely?

File: files/en-us/web/css/@scroll-timeline/index.md [hunk 2]
```diff
@@ -166,6 +166,13 @@ We create an `@scroll-timeline` called `squareTimeline`, setting the `source` as
...
 
+## Specifications
+
+This feature was in a draft specification and has been removed from it. It is no longer on track to become a standard.
+
+## Browser compatibility
+
+No released browser ever implemented this feature.
```
Comment (teoli2003) [inline files/en-us/web/css/@scroll-timeline/index.md:175]:
```suggestion
No released browser ever implemented this feature.

```

**Comment by teoli2003** [comment]
> If no browser ever implemented this and the specification was dropped, why don't we drop this page entirely?

I'm unsure: on one side, yes; on the other, this is still an experimental feature on Firefox…

I wish we could keep it and "hide" it in case it comes back in the future.

**Comment by queengooborg** [comment]
> this is still an experimental feature on Firefox

Ah, sorry, I glanced over that in the compat data (I misread a comment on the related BCD PR as "this is yet to be implemented in Firefox").  Since Firefox _does_ support the feature behind a flag, then I think that retaining the BCD data/tables would be better to show there is a browser with experimental, flagged implementation.  I think we're more likely to catch the removal from the BCD side.

**Comment by sideshowbarker** [comment]
> If no browser ever implemented this and the specification was dropped, why don't we drop this page entirely?

In general I wish we would do that in all cases like this one — and have tried to in other cases in the past; but whenever I’ve tried, it seems like the decision we’ve ended up (re)reaching is that we seem to have a de facto practice of keeping things like this around, once they are in.

So IMHO part of the lesson on features like this one is that we really would be better off not creating content for them to begin with — because the cost in time of removing them once they are created seems to be greater than the cost of creating them.

> I wish we could keep it and "hide" it in case it comes back in the future.

Since the source is under version control, `git rm`’ing it pretty much exactly has the effect of hiding it such as way that we can retrieve/restore it back from version control whenever we want.

**Comment by teoli2003** [comment]
@Rumyra: is there any plan to launch this in Firefox or is it just sitting as "experimental" but with no goal of shipping it anymore?

**Comment by teoli2003** [comment]
Friendly ping @Rumyra and @jpmedley here.

**Comment by Rumyra** [comment]
Thanks @teoli2003 - I've been going back and forth on this one :)

Let's not block this pr - the changes @sideshowbarker have made are good. From small tests it seems it was in Chrome behind a flag as well.

I'd also be happy to remove this page (as the feature wasn't around very long before changing) - but it'd be nice to have a good redirect for it, so maybe something for when the new proposed way of scroll animations comes into browsers.

**Review by teoli2003 — APPROVED** [review]
Let's merge this then. If needed, follow-up PRs are welcome.

**Pull request merged**